resources/testdata/data/cldr/cldr-dates-full/main/en-CA/timeZoneNames.json

```diff
@@ -1144,9 +1144,6 @@
               "Broken_Hill": {
                 "exemplarCity": "Broken Hill"
               },
-              "Currie": {
-                "exemplarCity": "Currie"
-              },
```

Comment:
stale.

Reply:
I think this is a change from 38 to 39 on a file that was never updated to 39.0.0-BETA2 within the ICU4X repo.